add-domain

```diff
@@ -999,3 +999,32 @@ wiflix.xyz
 wiflix.yachts
 coronavirus.zone
 fullcopec.zone
+dexsceerner.net
+app.dexscreener-home.net
+dexscreener-home.net
+traolerjoe.org
+traderjoexyz.bylfg.org
```

Comment:
Double of `bylfg.org`

- https://kb.mypdns.org/issue/MTX-127020/bylfg.org MTX-127020
- https://mypdns.youtrack.cloud/issue/MTXB-209538/bylfg.org MTXB-209538
- https://github.com/mypdns/matrix/issues/127020